src/mcts/search.cc

```diff
@@ -119,6 +119,26 @@ class MEvaluator {
     return m;
   }
 
+  float GetM(Node* child, float q) const {
+    if (!enabled_ || !parent_within_threshold_) return 0.0f;
+    const float child_m = child->GetM();
+    float m = std::clamp(m_slope_ * (child_m - parent_m_), -m_cap_, m_cap_);
+    // Microsoft compiler does not have a builtin for copysign and emits a
+    // library call which is too expensive for a hot path like this.
+#if defined(_MSC_VER)
```

Comment:
May make sense to have an inline function in `util/fastmath.h` or create `util/misc.h` for such cases.

Reply:
true - this is a copy/paste of code immediately above.